pgmoon/socket.moon

```diff
@@ -93,6 +99,8 @@ create_luasocket = do
         ngx.socket.tcp!
       when "luasocket"
         create_luasocket!
+      when "haproxy"
+        create_luasocket("haproxy")
```

Comment:
@leafo using your `create_luasocket` interface now instead, just using the raw `core.tcp` wasn't playing well with the expected socket API.